python/lsst/ts/eas/eas_csc.py

```diff
@@ -84,290 +88,238 @@ def __init__(
             simulation_mode=simulation_mode,
             override=override,
         )
-        self.eas = None
-        self.m1m3ts = salobj.Remote(self.domain, "MTM1M3TS")
-        self.ess = salobj.Remote(self.domain, "ESS", index=112)
 
-        # Variables for the m1m3ts loop
-        self.m1m3_thermal_task = utils.make_done_future()
-        self.enabled_event = (
-            asyncio.Event()
-        )  # An event that is set when the CSC is enabled.
-        self.control_loop_lock = asyncio.Lock()
-
-        self.heater_demand: list[int] = [0] * 96
-        self.fan_demand: list[int] = [30] * 96
-        self.temperature_target_offset = -1.0
+        self.last_vec04_time: float = (
+            0  # Last time VEC-04 was changed (UNIX TAI seconds).
+        )
+        self.health_monitor_task = utils.make_done_future()
+        self.wind_history = pd.DataFrame(
+            {
+                "speed": pd.Series(dtype=float),
+                "private_sndStamp": pd.Series(dtype=float),
+            }
+        )
 
-        self.old_valve_position: float = float("nan")
+    async def handle_summary_state(self) -> None:
+        """Override of the handle_summary_state function to
+        set up the control loop.
+        """
+        self.log.debug(f"handle_summary_state {salobj.State(self.summary_state).name}")
 
-        self.log.info("__init__")
+        if self.disabled_or_enabled:
+            if self.health_monitor_task.done():
+                self.health_monitor_task = asyncio.create_task(self.health_monitor())
 
-    async def connect(self) -> None:
-        """Connect the EAS CSC or start the mock client, if in
-        simulation mode.
-        """
-        self.log.info("Connecting")
-        self.log.info(self.config)
-        self.log.info(f"self.simulation_mode = {self.simulation_mode}")
-        if self.config is None:
-            raise RuntimeError("Not yet configured")
-        if self.connected:
-            raise RuntimeError("Already connected")
-        if self.simulation_mode == 1:
-            # TODO Add code for simulation case, see DM-26440
-            pass
         else:
-            # TODO Add code for non-simulation case
+            await self.health_monitor_shutdown()
+
+    async def health_monitor_shutdown(self) -> None:
```

Comment:
It is a good practice to start method names with a verb, to indicate the "action" being executed. Consider renaming this `shutdown_health_monitor`.

Also, please, add a docstring.